### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by Ryxx** [zero01101/openOutpaint#249]
### Is your feature request related to a problem? Please describe.

Problem: Having to organize and move the current floating panels, because they get in the way of the canvas.

### Describe the solution you'd like

A solution would be to dock these panels once and for all (as an option so those who prefer floating panels can keep them) so we have a clean UI.

### Proposed workflow

That option would to dock the floating panels at the sides in full columns (without padding from the borders) so they don't get in the way by hovering over the generative canvas and we don't have to move / expand them all the time. And the canvas could be in its own window so it doesn't get covered by the panels when zooming.

### Describe alternatives you've considered

It could look like this: the "Openpaint panel" in a left column, the vertical "Toolbar" immediately at its right (going from top to bottom, so it's not floating, like in Photoshop), the "History" and "Layers" panels to be split in 2 vertically in a column at the right.

![Capture](https://github.com/zero01101/openOutpaint/assets/46426250/89fdabd3-53b6-4fb2-a4ec-e8e76a0b5050)

Bonus: also if the panels' backgrounds could have the same gray color as the toolbar and the various boxes to have less colors and a more unified UI, it would look even more professional.

Thanks.

**Comment by zero01101** [comment]
ok so i'm certainly not averse to that being an option but uhhh i suspect that's gonna take some PRs from better programmers than myself lol; the state of openOutpaint before i got UI enhancement PRs was not attractive and speaks volumes of my frontend capability :P 
![lol old openoutpaint was really ugly](https://github.com/zero01101/openOutpaint/assets/1649724/87993a94-85d7-4507-ab44-5f008759b5ac)

**Comment by Ryxx** [comment]
Haha, well sure the GUI looks a bit simplistic but at least it was docked ;)
Is it about lacking the coding knowledge to implement such UI or is it just about designing it?
Because from what I see, you already have a solid base and it would be just a matter of color and font fiddling and a little bit of rearranging the boxes / widgets…

**Comment by zero01101** [comment]
>Is it about lacking the coding knowledge 

yep that's the one lol, i'm pretty dumb :D most of the UI/UX stuff is definitely other folks' work.  i can generally figure stuff out given enough time, this one's just probably not going to be particularly high priority, especially while SDXL outpainting sucks as bad as it does grumble grumble
